### PR TITLE
Improve kotlin api generator to provide replacement for a deprecated method

### PR DESCRIPTION
**Pull request by sokomishalov**
+ minor typo fix.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,8 +60,8 @@
         <junit5.version>5.7.1</junit5.version>
         <hamcrest-library.version>2.2</hamcrest-library.version>
         <hdr-histogram-version>2.1.12</hdr-histogram-version>
-        <kotlin.version>1.4.32</kotlin.version>
-        <kotlinx-coroutines.version>1.4.3</kotlinx-coroutines.version>
+        <kotlin.version>1.5.10</kotlin.version>
```
Comment (mp911de) [inline pom.xml:63]: Let's stick with Kotlin 1.4 for the time being. Also, please move the version change into its own commit.

Comment (sokomishalov) [reply pom.xml]: Sure
I've reverted this change

**Comment by mp911de** [comment]
Thank you for your contribution. That's merged and backported now.
